src/pages/markets/NewMarket.tsx

```diff
@@ -0,0 +1,249 @@
+              .div(1e18)
+              .toFixed(isMainnet ? 0 : 18),
+            NATIVE_TOKEN_DENOM: chainTokenLabel,
+          },
+        }),
+      },
+    ];
+  }, [stringGetter, newMarketProposal, chainTokenLabel]);
+
+  return (
+    <Styled.Page>
+      <Styled.HeaderSection>
+        <Styled.ContentSectionHeader
+          title={stringGetter({ key: STRING_KEYS.SUGGEST_NEW_MARKET })}
+          slotRight={
+            <IconButton iconName={IconName.Close} onClick={() => navigate(AppRoute.Markets)} />
+          }
+          subtitle={isNotTablet && stringGetter({ key: STRING_KEYS.ADD_DETAILS_TO_LAUNCH_MARKET })}
+        />
+      </Styled.HeaderSection>
+      <Styled.Content>
+        <div>
+          <Button
+            slotLeft={<Styled.Icon iconName={displaySteps ? IconName.Hide : IconName.HelpCircle} />}
+            onClick={() => setDisplaySteps(!displaySteps)}
+          >
+            {displaySteps
+              ? stringGetter({ key: STRING_KEYS.HIDE_STEPS })
+              : stringGetter({ key: STRING_KEYS.SHOW_STEPS })}
+          </Button>
+          {displaySteps && (
+            <>
+              <h2>{stringGetter({ key: STRING_KEYS.STEPS_TO_CREATE })}</h2>
+              {steps.map((item) => (
+                <StepItem
+                  key={item.step}
+                  step={item.step}
+                  title={item.title}
+                  subtitle={item.subtitle}
+                />
+              ))}
+            </>
+          )}
+        </div>
+        <Styled.FormContainer>
+          <NewMarketForm />
+        </Styled.FormContainer>
+      </Styled.Content>
+    </Styled.Page>
+  );
+};
+
+const Styled: Record<string, AnyStyledComponent> = {};
+
+Styled.Page = styled.div`
+  ${layoutMixins.contentContainerPage}
+  gap: 1.5rem;
+
+  > * {
+    --content-max-width: 80rem;
+    max-width: min(calc(100vw - 4rem), var(--content-max-width));
+  }
+
+  @media ${breakpoints.tablet} {
+    --stickyArea-topHeight: var(--page-header-height-mobile);
+    padding: 0 1rem 1rem;
+
+    > * {
+      max-width: calc(100vw - 2rem);
+      width: 100%;
+    }
+  }
+`;
+
+Styled.ContentSectionHeader = styled(ContentSectionHeader)`
+  @media ${breakpoints.tablet} {
+    padding: 1.25rem 1.875rem 0;
+
+    h3 {
+      font: var(--font-extra-medium);
+    }
+  }
+`;
+
+Styled.HeaderSection = styled.section`
+  ${layoutMixins.contentSectionDetached}
+
+  @media ${breakpoints.tablet} {
+    ${layoutMixins.flexColumn}
+    gap: 1rem;
+
+    margin-bottom: 0.5rem;
+  }
+`;
+
+Styled.Content = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 2rem;
+  margin: 0 auto;
+
+  @media ${breakpoints.tablet} {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    margin: 0 auto;
+  }
+
+  h2 {
+    font: var(--font-large-medium);
+    color: var(--color-text-2);
+    margin: 1rem;
+  }
+`;
+
+Styled.Icon = styled(Icon)`
+  width: 1.5rem;
```

Comment:
if we switch size i think you need to change font-size, otherwise for margin can you just put `margin-right: 0.5ch`?